modules/tinymce/src/core/main/ts/SelectionOverrides.ts

```diff
@@ -160,6 +160,17 @@ const SelectionOverrides = (editor: Editor): SelectionOverrides => {
       }
     });
 
+    editor.on('focusin', (e) => {
+      if (fakeCaret.isShowing() && !NodeType.matchNodeNames([ 'body' ])(e.target) && !editor.dom.isEditable(e.target.parentNode)) {
```

Comment:
It's safer to do a `e.target == editor.getBody()` since for inline mode `body` is not the name of the root editor container.

- [ ] Change editor root container check